lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -167,6 +183,35 @@ WasmBinaryReader::ReadFromModule()
      return wnLIMIT;
 }
 
+SectionCode
+WasmBinaryReader::SectionHeader()
+{
+    UINT len;
+    UINT sectionSize;
+    UINT idSize;
+
+    sectionSize = LEB128(len);
+    m_moduleState.count += len;
+    idSize = LEB128(len);
+    m_moduleState.count += len;
+    const char *sectionName = (char*)(m_pc);
+    m_pc += idSize;
+    m_moduleState.count += idSize;
+
```

Comment:
Ah, right. This shouldn't be updating count at all. However, we can't set the m_moduleState.size just yet because this size is a byte size of a section (exclusive of the varint size itself) and not a module section element count, which will follow the section name usually. We'll need a separate piece of state to track section byte size eventually.